src/predicates/pathSatisfies.js

```diff
@@ -0,0 +1,61 @@
+/** @license ISC License (c) copyright 2019 original and current authors */
+/** @author Ian Hofmann-Hicks (evilsoft) */
+
+const curry = require('../core/curry')
+const isArray = require('../core/isArray')
+const isEmpty  = require('../core/isEmpty')
+const isInteger = require('../core/isInteger')
+const isNil = require('../core/isNil')
+const isPredOrFunc = require('../core/isPredOrFunc')
+const isString = require('../core/isString')
+const predOrFunc = require('../core/predOrFunc')
+
+const err = name =>
+  `${name}: First argumnet must be an Array of Non-empty Strings or Integers`
```

Comment:
mis-spelling of argument, this is the start of it and there is a bunch of copy-🍝 below.